### PR TITLE
Cleanup and documentation

### PR DESCRIPTION
**Pull request by DALove1025**
Several little improvements to clean up the code. In particular:

* removing unused files;
* removed some unnecessary svg elements from a few svg files;
* moving a few methods around in advance of some deeper refactoring;
* added a cypress test of the "labels" check-box;
* isolated the seeded random number generation wrapper, `rand()` into it's own file for use in another project;
* added a unit test for `rand()`; and,
* clamping the data supplied in the river data store utility to be in the range 0..100.

### CODE REVIEW

File: src/components/picture/picture-area.tsx [hunk 1]
```diff
@@ -162,6 +148,17 @@ const buildingsAgriburg: IBuilding[] = [
...
+//   public render() {
+//     const { x, y, w, h, f } = this.props;
+//     return(
+//       <div style={innerStyle}>
+//         <Dam width={25 * f} height={20 * f} transform={`translate: ${x * f} ${y * f}`} />
+//       </div>
+//     );
+//   }
+// }
+
```
Comment (dstrawberrygirl) [inline src/components/picture/picture-area.tsx]: Not sure this block should be here?

File: src/components/picture/picture-area.tsx [hunk 2]
```diff
@@ -574,18 +565,19 @@ export class PictureArea extends BaseComponent<IProps, {}> {
       <div className="subsection simulation">
         <div className="picture-area-container">
           { renderScenery() }
-          { renderTrees() }
           { renderRivers(flowPercentage / 25) }
           { renderLake(currentLakeArea) }
           { renderDamn() }
+
+          {/* <FullSizeThing x={200} y={250} e_x={20} e_y={25} w={600} h={340} f={width / 600.0}/> */}
```
Comment (dstrawberrygirl) [inline src/components/picture/picture-area.tsx]: and this looks like it needs to come out

Comment (DALove1025) [reply src/components/picture/picture-area.tsx]: Yep, I missed it. Thanks.

File: src/components/picture/picture-area.tsx (hunk 1 above)
Comment (DALove1025) [reply src/components/picture/picture-area.tsx]: Indeed, it should not be there.

**Pull request merged**